data/hai/hai reveal 1 intro.txt

```diff
@@ -315,8 +316,8 @@ mission "Trader Directions to Hai Space"
 		conversation
 			`Arriving at the only place you can think of where you feel sure you can find out more information, you find that <origin>, like most places, is abuzz with the recent news. Avoiding the many large, cheap, and boisterous establishments that service the many crews and workers of the Megaparsec corporate machinery here, you make your way to one of the marginally more up-market places, cunningly marketed as a merchant-only dive.`
 			`	Since the story first broke, there has been an endless cycle of contradictory reporting on the matter plastered across most screens in major thoroughfares. Here, however, it is remarkably absent, with only one of the screens on display covering the "violent alien race" at all - running stats on how many different versions of the story are now being told.`
-			`	Casting around the room, you pick out a merchant who looks like she's been around the block a few times. In addition to looking like she could pick up most people and break them over her knee, she's conspicuous by the threatening glare you watch her give a younger captain who left a table talking about the Hai to approach her... briefly. After the captain is briskly sent away, you make your way over. Her glare falters in a flash of recognition before subsiding to a reserved scowl.`
-			`	She waits until you're directly next to her table. "The famous Captain <last>. Which ancient Sumerian deity did I offend to be granted the gift of your attention, hey?"`
+			`	Casting your eyes around the room, you pick out a merchant who looks like she's been around the block a few times, and looks like she could pick up most people and break them over her knee. She's especially conspicuous with her threatening glare that you watch her give a younger captain, who left a table talking about the Hai to approach her... briefly. After the captain is briskly sent away, you make your way over. Her glare falters in a flash of recognition before subsiding to a reserved scowl.`
```

Comment:
```suggestion
			`	Casting your eyes around the room, you pick out a merchant who looks like she's been around the block a few times and also like she could pick up and break most people over her knee. Especially conspicuous is the threatening glare that you watch her give a younger captain when they leave a table talking about the Hai to approach her... briefly. After the captain is briskly brushed aside, you make your way over. Her glare falters in a flash of recognition before subsiding to a reserved scowl.`
```

This is still more than a bit awkward, but headed in the right direction.

Reply:
Yes, it will have to be looked at later, but it's been changed to your suggestion for now.